BEST_PRACTICES.md

```diff
@@ -121,85 +138,60 @@ The following are options that can be passed into the httpMiddlewareOption when
-  ...
-  .build()
-```
+In the Commercetools Typescript SDK tokens are automatically fetched by default using the client credentials flow and injected into the request header.
+When a request is made, if the token in the header is expired or unavailable, the auth-middleware checks for this and automatically calls the platform to generate a new token and inject it into the header.
+Therefore, no action is required by the customer to handle tokens when using the TS SDK.
 
 ## Customizing Responses
 
-The response data gotten from a request can be customized to include certain details and also exclude others. In the JS/TS SDK the `originalRequest` sent to the SDK is included in the response body by default, however, if this behavior is not wanted, they can easily be excluded from both the success and error responses.
+The response data gotten from a request can be customized to include or exclude certain details. By default, the Commercetools JS/TS SDK includes the `originalRequest` in the response body.
+However, if this behavior is not wanted, they can easily be excluded from both the success and error responses.
+
+To customize the response data, configure the `httpMiddlewareOptions` when initializing the client (see [Client Initialization](#client-initialization)).
 
 This can easily be accomplished in a simple and straightforward way, see example below.
 
 ```ts
 const httpMiddlewareOptions = {
-  host: 'https://api.europe-west1.gcp.commercetools.com',
-  excludeRequestInErrorResponse: true,
+  host: process.env.CTP_API_HOST,
+  includeRequestInErrorResponse: true,
   includeOriginalRequest: true,
-  ...fetch,
+  httpClient: fetch,
 }
 ```
 
-The `maskSensitiveHeaderData` option redacts sensitive header information like the authorization header bearing the platform token and is enabled by default. The `excludeRequestInErrorResponse` excludes the `originalRequest` information from the response body of the **error response** while `includeOriginalRequest` includes the original client request in the **success response**.
+The `maskSensitiveHeaderData` option redacts sensitive header information like the authorization header bearing the platform token and is enabled by default. The `includeRequestInErrorResponse` excludes the `originalRequest` information from the response body of the **error response** while `includeOriginalRequest` includes the original client request in the **success response**.
 
 ## Client Reuse
 
 The client can be created once and reused throughout the application by creating the `apiRoot` object instance and reuses the object instance throughout the application.
 
 ```ts
 // client.ts
-import ClientBuilder from '@commercetools/sdk-client-v2'
+import ClientBuilder from '@commercetools/ts-client'
 import createApiBuilderFromCtpClient from '@commercetools/platform-sdk'
-...
+//...
 
 const clientObject = new ClientBuilder()
 export default clientObject
 
 // apiroot.ts
 import clientObject from '../apiroot.ts'
-...
+
```

Comment:
```suggestion
...
```

Reply:
I believe that just this it's better to be commented otherwise can be confused with Spread operator or Rest parameters.
So I am gonna add back `/...`